src/coreclr/vm/jitinterface.cpp

```diff
@@ -9794,59 +9755,178 @@ CorInfoHFAElemType CEEInfo::getHFAType(CORINFO_CLASS_HANDLE hClass)
     return result;
 }
 
+namespace
+{
+    CorInfoCallConvExtension getUnmanagedCallConvForSig(Module* mod, PCCOR_SIGNATURE pSig, DWORD cbSig, bool* pSuppressGCTransition)
+    {
+        SigParser parser(pSig, cbSig);
+        ULONG rawCallConv;
+        if (FAILED(parser.GetCallingConv(&rawCallConv)))
+        {
+            COMPlusThrowHR(COR_E_BADIMAGEFORMAT);
+        }
+        switch ((CorInfoCallConv)rawCallConv)
+        {
+        case CORINFO_CALLCONV_DEFAULT:
+            _ASSERTE_MSG(false, "bad callconv");
+            return CorInfoCallConvExtension::Managed;
+        case CORINFO_CALLCONV_C:
```

Comment:
This can use `IMAGE_CEE_CS_CALLCONV_C`, etc. 

`IMAGE_CEE_CS_CALLCONV_C` is for the raw metadate encodings that we are reading here, so it would be more appropriate here.

`CORINFO_CALLCONV_C` is JIT/EE interface abstraction that happens to use same values.